public/assets/meetups.json

```diff
@@ -1,3 +1 @@
-[
-  "./assets/meetups/2024-04-11-lmt-01.json"
-]
+["./assets/meetups/2024-04-11-lmt-01/data.json", "./assets/meetups/2024-09-11-lmt-02/data.json"]
```

Comment:
```suggestion
[
    "./assets/meetups/2024-04-11-lmt-01/data.json",
    "./assets/meetups/2024-09-11-lmt-02/data.json",
]
```

Reply:
I think this was caused by prettier, and my set line width. I changed the `printWidth` value from 100 to 80 to fix this problem, this will extend the files by couple of lines but make them more readable.